<launch>
    <node name="camera1" pkg="usb_cam" type="usb_cam_node" output="screen" >
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="mjpeg" />
        <param name="camera_frame_id" value="base_camera" />
    </node>

    <group ns="camera1">
        <node pkg="image_proc" type="image_proc" name="image_proc_node" output="screen">
        </node>
    </group>

</launch>
